packages/schematics/src/jest/templates/jest/project/jest.config.js

```diff
@@ -0,0 +1,13 @@
+// https://github.com/thymikee/jest-preset-angular#brief-explanation-of-config
+module.exports = {
```

Comment:
The configuration can be shortened.
`@angular-builders/jest` takes care of using the right preset.

See: https://github.com/meltedspark/angular-builders/blob/master/packages/jest/src/jest-config/default-config.ts

We still can apply `moduleNameMapper` & `transformIgnorePatterns`.

```js
const esModules = ['jest-test'].join('|');

module.exports = {
 moduleNameMapper: {
    '@app/(.*)': '<rootDir>/src/app/$1',
    '@assets/(.*)': '<rootDir>/src/assets/$1',
    '@core/(.*)': '<rootDir>/src/app/core/$1',
    '@env': '<rootDir>/src/environments/environment',
    '@src/(.*)': '<rootDir>/src/src/$1',
    '@state/(.*)': '<rootDir>/src/app/state/$1',
    // This is part of the default config of @angular-builders/jest
    '\\.(jpg|jpeg|png)$': `${__dirname}/mock-module.js`
  },
  transformIgnorePatterns: [`<rootDir>/node_modules/(?!${esModules})`]
}
```